Sample Applications/DataBindingDemo/Styles.xaml

```diff
@@ -116,6 +116,18 @@
         <Setter Property="Width" Value="500" />
     </Style>
 
+    <Style x:Key="AuctionItemStyle">
```

Comment:
Call it `AuctionItemContainerStyle` instead, to emphasize that it's applied to the item-container.